client/Client.d.ts

```diff
@@ -0,0 +1,81 @@
+// There is no Interger in Typescript so it has been classed as a number but the Interger is still expected. 
+type Interger = number;
+
+declare class Client {
+  hostUrl: any;
+  apiKey: any;
```

Comment:
If `this.host.URL = host` does that make `hostURL` a string? and same goes for `apiKey`